eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -28,6 +28,8 @@ steps:
       Write-Host "##vso[task.setvariable variable=SKIP_CREDSCAN]true"
     }
   displayName: CredScan setup
+  condition: succeededOrFailed()
```

Comment:
Do we want to run this if the pipeline failed? Another option would be to change the condition below to remove the failed part as well.

Reply:
We do. `Analyze` checks as a whole are kinda expected to fail, so any FURTHER checks should still run even if this one fails.